dev/create-release/spark-rm/Dockerfile

```diff
@@ -38,7 +42,8 @@ ARG PIP_PKGS="pyopenssl pypandoc numpy pygments sphinx"
 #
 # This is all in a single "RUN" command so that if anything changes, "apt update" is run to fetch
 # the most current package versions (instead of potentially using old versions cached by docker).
-RUN echo 'deb http://cran.cnr.Berkeley.edu/bin/linux/ubuntu xenial/' >> /etc/apt/sources.list && \
+RUN apt-get update && $APT_INSTALL gnupg && \
```

Comment:
18.04 doesn't have gpg? I can see why here but nit all other places we are apt-get clean after
(though maybe unnecessary in a build image)

Reply:
gpg is not installed by default in 18.04, and I was surprised.